Dungeon_of_Munchkinomicon.py

```diff
@@ -63,11 +71,11 @@ def room1(name):
 			print line
 			print "You raise your katana, unsure of what lies behind the red door."
 			print "A rabid yorkshire terror attacks! %s uses Katana. It's super effective!" % name
-			reddoor(name)
+			redDoor(name)

```

Comment:
In most programming languages you want to name functions and variables with some form of [CamelCase](http://en.wikipedia.org/wiki/CamelCase) or [snake_case](http://en.wikipedia.org/wiki/Snake_case) or some variation (I chose the mixedCase variation in this change which is popular with javascript). After making this commit, I did some research and it looks like snake_case is what is popular for python functions. So I should rename this function again to red_door.

The way I did the research was I looked at the source code for flask which is a way to build websites with python. All their functions were snake_case. [Here's an example from the flask source code.](https://github.com/mitsuhiko/flask/blob/master/flask/helpers.py#L137)